docs/workers.md

```diff
@@ -205,8 +205,8 @@ information.
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/context/.*$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/members$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/state$
-    ^/_matrix/client/(v1|unstable/org.matrix.msc2946)/rooms/.*/hierarchy$
-    ^/_matrix/client/(v1|unstable/org.matrix.msc2716)/rooms/.*/batch_send$
+    ^/_matrix/client/v1/rooms/.*/hierarchy$
+    ^/_matrix/client/unstable/org.matrix.msc2716/rooms/.*/batch_send$
```

Comment:
This was wrong, see https://github.com/matrix-org/synapse/pull/12868#discussion_r881542588